user-guide/Standard_Apps/Monitoring_Solutions/META_WhatsApp_Messaging/Using_META_WhatsApp_Messaging.md

```diff
@@ -0,0 +1,5 @@
+---
+uid: Using_META_WhatsApp_Messaging
+---
+
+# Using the META WhatsApp Messaging solution
```

Comment:
Could you complete this page on how to use the solution after finishing with the installation process, @SebCuellar? Currently, the documentation for the META WhatsApp Messaging solution only includes a summary of its purpose on the introduction page.

Reply:
At the end of the solution's installation guide, I added a link to the **connector documentation**, which describes how to use it with the solution. I did this to avoid having redundant information in different places. Maybe if I change the last title to _"Connector Installation & Usage"_ it would be more intuitive and understandable. What do you think?
Also, I'm updating the connector documentation for the same purpose of not having redundant information, but I need to publish this solution documentation to include references to it.